src/main/kotlin/org/wycliffeassociates/otter/common/domain/resourcecontainer/FileUsfmExtensions.kt

```diff
@@ -0,0 +1,39 @@
+package org.wycliffeassociates.otter.common.domain.resourcecontainer
+
+import org.wycliffeassociates.otter.common.collections.tree.Tree
+import org.wycliffeassociates.otter.common.collections.tree.TreeNode
+import org.wycliffeassociates.otter.common.data.model.Collection
+import org.wycliffeassociates.otter.common.data.model.Content
+import org.wycliffeassociates.otter.common.domain.usfm.ParseUsfm
+import java.io.File
+import java.io.IOException
+
+fun File.parseUSFMToChapterTrees(projectSlug: String): List<Tree> {
+    if (extension != "usfm")
+        throw IOException("Not a USFM file")
```

Comment:
my preference honestly is to always add { } around the if block

If not that, at the very least make the throw be on the same line as the if